### PR TITLE
LPS-170159 Replace PUT request with PATCH request

### PR DESCRIPTION
**Pull request by devpedrotavares**
[LPS-170159](https://issues.liferay.com/browse/LPS-170159) 
Did the fix discussed in [this slack thread](https://liferay.slack.com/archives/C026NQPV6TV/p1673972360765639)

### CODE REVIEW

File: modules/apps/object/object-rest-impl/rest-openapi.yaml [hunk 1]
```diff
@@ -219,6 +219,32 @@ paths:
...
+            responses:
+                200:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/ObjectEntry"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/ObjectEntry"
+            tags: ["ObjectEntry"]
```
Comment (carolmariaabb) [inline modules/apps/object/object-rest-impl/rest-openapi.yaml:247]: Hi @devpedrotavares 

I believe we also need to add one to `"/scopes/{scopeKey}/by-external-reference-code/{externalReferenceCode}":` ([line 294](https://github.com/liferay-objects/liferay-portal/blob/eec7a82dd5ae372dae02d5a2844b56c8cb6cad97/modules/apps/object/object-rest-impl/rest-openapi.yaml#L294)) for objects scoped by site. Could you please check? Thanks!

image

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-rest-impl/rest-openapi.yaml (hunk 1 above)
Comment (devpedrotavares) [reply modules/apps/object/object-rest-impl/rest-openapi.yaml:247]: Hi @carolmariaabb,
that is right! We needed to add this endpoint. I also had to override the generated patch to make it call the proper methods.

**Review by carolmariaabb — APPROVED** [review]
LGTM!

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
@devpedrotavares please fix the SF failure

**Comment by carolmariaabb** [comment]
ci:stop

**Comment by devpedrotavares** [comment]
> @devpedrotavares please fix the SF failure

just fixed it!

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:test:object

**Comment by devpedrotavares** [comment]
sent directly to Brian: https://github.com/brianchandotcom/liferay-portal/pull/129225

**Comment by devpedrotavares** [comment]
ci:close

**Pull request closed without merging**